packages/fiori/src/DynamicPage.ts

```diff
@@ -44,24 +45,36 @@ const SCROLL_DEBOUNCE_RATE = 0; // ms
 })
 
 class DynamicPage extends UI5Element {
-	static i18nBundle: I18nBundle;
-
-	constructor() {
-		super();
-
-		this._updateMediaRange = this.updateMediaRange.bind(this);
-	}
-
-	static async onDefine() {
-		DynamicPage.i18nBundle = await getI18nBundle("INIT_PACKAGE_VAR_NAME");
-	}
-
+	/**
+	 * Defines if the header is snapped.
+	 *
+	 * @type {boolean}
```

Comment:
type and name annotations are not needed anymore since the merge of the new jsdoc annotation rules https://github.com/SAP/ui5-webcomponents/pull/7613